app/services/editions/db/IssueQueries.scala

```diff
@@ -201,42 +194,22 @@ trait IssueQueries extends Logging {
       LEFT JOIN cards ON (cards.collection_id = collections.id)
       WHERE edition_issues.id = $id
       """.map { rs =>
-          EditionsIssue.fromRow(rs).toOption.map { issue => GetIssueRow(
-            issue,
-            DbEditionsFront.fromRowOpt(rs, "fronts_"),
-            DbEditionsCollection.fromRowOpt(rs, "collections_"),
-            DbEditionsCard.fromRowOpt(rs, "cards_"))
-        }
+        val maybeIssue = EditionsIssue.fromRow(rs).toOption
+        val frontRow = FrontAndNestedEntitiesRow(
+          DbEditionsFront.fromRowOpt(rs, "fronts_"),
+          DbEditionsCollection.fromRowOpt(rs, "collections_"),
+          DbEditionsCard.fromRowOpt(rs, "cards_")
+        )
+
+        maybeIssue.map(issue => (issue, frontRow))
```

Comment:
Intrigued why this is now needed? (not saying it _isn't_, just intrigued!)

Reply:
IIUC, this is related to 

> I've factored out the nested row aggregation logic common to issue and fronts queries in https://github.com/guardian/facia-tool/commit/dd57774336a9b7bba5066d153e1e3fd98514d354

in that `GetIssueRow` is replaced by `FrontAndNestedEntitiesRow` (decoupling the issue data in this ResultSet from the Fronts data), so we can use `FrontAndNestedEntitiesRow` elsewhere – does that help to explain? May have wrong end of stick!